packages/lib/src/components/Card/components/CardInput/components/StoredCardFieldsWrapper.tsx

```diff
@@ -0,0 +1,40 @@
+import { h } from 'preact';
+import LoadingWrapper from '../../../../internal/LoadingWrapper';
+import StoredCardFields from './StoredCardFields';
+import { getInstallmentsComp } from './Installments/GetInstallmentsComp';
+
+export const StoredCardFieldsWrapper = ({
+    sfpState,
+    setFocusOn,
+    cvcPolicy,
+    focusedElement,
+    hasInstallments,
+    handleInstallments,
+    showAmountsInInstallments,
+    ...props
+}) => {
```

Comment:
It would be nice to define a proper interface for the `Props` of these new components, narrowing down the types for the components. 

I don't know how much work that is though; and how that could be reused accross this wrapper, the Card wrapper, and any other Card component that share these props.

Reply:
I agree. I've always thought it's very _opaque_ just sending through all the props - it's not clear what is actually needed further down the line in the card/securedField components and what is just unnecessary "baggage".

I tried, some time ago, to list all the props in the `SFPProps` interface in `/internal/SecuredFields/defaultProps.ts` - and there are a lot of them (and this list must now be out of date). 

But I don't know if trying to isolate them all and pass them through, individually, to the card related components is worth it or not? Will it just add length to the file that I've just managed to shave off? Should we just embrace the spread operator and let it work its opaque magic?

It might be worth looking into... in another PR.